lib/data_loader.rb

```diff
@@ -0,0 +1,31 @@
+require 'csv'
+require_relative 'game'
+require_relative 'team'
+require_relative 'game_team'
+require_relative './game_processor'
+require_relative './league_processor'
+require_relative './season_processor'
+require_relative './team_processor'
+
+class DataLoader
```

Comment:
I love this idea of a parent class for our data fetching methods that were previously taking up seemingly unnecessary real estate